beacon_node/beacon_chain/tests/block_verification.rs

```diff
@@ -1810,3 +1829,14 @@ async fn import_execution_pending_block<T: BeaconChainTypes>(
         }
     }
 }
+
+fn get_cgc<E: EthSpec>(blobs_opt: &Option<DataSidecars<E>>) -> usize {
+    if let Some(data_sidecars) = blobs_opt.as_ref() {
+        match data_sidecars {
+            DataSidecars::Blobs(b) => b.len(),
```

Comment:
CGC is not relevant for blobs, might as well just return 0?

Reply:
2ba26543e5f039b1ee9774bb1c4c2f21dfa2a5e1